src/main/java/frc/robot/subsystems/climber/Climber.java

```diff
@@ -0,0 +1,122 @@
+package frc.robot.subsystems.climber;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonFX;
+import edu.wpi.first.wpilibj.Solenoid;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+import frc.robot.Ports;
+import frc.robot.subsystems.UnitModel;
+
+public class Climber extends SubsystemBase {
+    private final TalonFX master = new TalonFX(Ports.Climber.MASTER);
+    private final TalonFX slave = new TalonFX(Ports.Climber.SLAVE);
+    private final UnitModel unitModel = new UnitModel(Constants.Climber.TICKS_PER_METER);
+    private final Solenoid stopper = new Solenoid(Ports.Climber.STOPPER);
+    private final Solenoid gearboxShifter = new Solenoid(Ports.Climber.GEARBOX_SHIFTER);
+
+    public Climber() {
+        slave.follow(master);
+
+        master.setInverted(Ports.Climber.IS_MASTER_INVERTED);
+        slave.setInverted(Ports.Climber.IS_SLAVE_INVERTED);
+
+        master.setSensorPhase(Ports.Climber.IS_MASTER_SENSOR_PHASE_INVERTED);
+//        slave.setSensorPhase(Ports.Climber.IS_SLAVE_SENSOR_PHASE_INVERTED);
+
+
+        master.configMotionCruiseVelocity(Constants.Climber.CRUISE_VELOCITY, Constants.TALON_TIMEOUT);
+        master.configMotionAcceleration(Constants.Climber.ACCELERATION, Constants.TALON_TIMEOUT);
+
+        master.config_kP(0, Constants.Climber.kP, Constants.TALON_TIMEOUT);
+        master.config_kI(0, Constants.Climber.kI, Constants.TALON_TIMEOUT);
+        master.config_kD(0, Constants.Climber.kD, Constants.TALON_TIMEOUT);
+        master.config_kF(0, Constants.Climber.kF, Constants.TALON_TIMEOUT);
+
+
+    }
+
+    /**
+     * Get the climber's elevation relative to the ground.
+     *
+     * @return the climber's height [m].
+     */
+    public double getHeight() {
+        return unitModel.toUnits(master.getSelectedSensorPosition());
+    }
+
+    /**
+     * ManageClimb height meters up with the robot.
+     *
+     * @param height requested height to climb [m].
+     */
+    public void setHeight(double height) {
+        master.set(ControlMode.MotionMagic, unitModel.toTicks(height));
+    }
+
+    /**
+     * Get whether the stopper is engaged.

```

Comment:
```suggestion
     * Whether the stopper is engaged.

```

Reply:
The "get" is important. That's what the method does.